nsis/data/languages/Nsis.slaspec

```diff
@@ -4,104 +4,110 @@ define endian = little;
-:FileClose is opcode=0x36 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileOpen is opcode=0x37 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileWrite is opcode=0x38 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileRead is opcode=0x39 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileSeek is opcode=0x3a & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FindClose is opcode=0x3b & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FindNext is opcode=0x3c & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FindFirst is opcode=0x3d & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:WriteUninstaller is opcode=0x3e & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:LogText is opcode=0x3f & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:SectionSet is opcode=0x40 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:InstTypeSet is opcode=0x41 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:GetLabelAddr is opcode=0x42 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:GetFunctionAddr is opcode=0x43 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:LockWindow is opcode=0x44 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileWriteUTF16LE is opcode=0x45 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
-:FileReadUTF16LE is opcode=0x46 & arg1 & arg2 & arg3 & arg4 & arg5 & arg6 { }
+:InvalidOpcode is opcode=0x00 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Return is opcode=0x01 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Jmp is opcode=0x02 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Abort is opcode=0x03 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Quit is opcode=0x04 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Call is opcode=0x05 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:UpdateText is opcode=0x06 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Sleep is opcode=0x07 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:BringToFront is opcode=0x08 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:ChDetailsView is opcode=0x09 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:SetFileAttributes is opcode=0x0a ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:CreateDir is opcode=0x0b ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:IfFileExists is opcode=0x0c ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:SetFlag is opcode=0x0d ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:IfFlag is opcode=0x0e ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:GetFlag is opcode=0x0f ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:Rename is opcode=0x10 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:GetFullPathName is opcode=0x11 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:SearchPath is opcode=0x12 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:GetTempFilename is opcode=0x13 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:ExtractFile is opcode=0x14 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:DeleteFile is opcode=0x15 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
+:MessageBox arg1 arg2 arg3 arg4 arg5 is opcode=0x16 ; arg1 ; arg2 ; arg3 ; arg4 ; arg5 ; arg6 { }
```

Comment:
just curious: why MessageBox is different and requires 'arg1 arg2..." before "is opcode", and others don't?

Reply:
It's to show the parameters in Ghidra's UI. I didn't add all the others for now, because I need to go through the list and do them one by one since they don't all have 6 arguments. Should I wait for messageBox and put it in another PR with the rest?

Reply:
No, that's good, I just didn't know why it's different :)